src/library_openal.js

```diff
@@ -540,13 +550,13 @@ var LibraryOpenAL = {
         var lUpZ = listener.up[2];
 
```

Comment:
Looks like `inverseMagnitude` is only used in this function, so how about defining it around this location (before the first use), as a normal `function inverseMagnitude(x, y, z) { .. }`? (otherwise, on the AL object it is harder to optimize away)